### PR TITLE
 github comments fixed

### PR DESCRIPTION
**Pull request by Deepak-webelight**
- github comments fixed
- seperate jwt and bycrypt into utils file

### CODE REVIEW

File: server/utils/bcrypt.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { hash, compare, genSalt } from "bcryptjs";
+
+export const createHashPassword = async (password: string): Promise<string> => {
+  try {
```
Comment (anshs-webelight) [inline server/utils/bcrypt.ts]: no need to use a try-catch block, an error will handled by the parent catch block automatically

File: server/services/user.service.ts [hunk 2]
```diff
@@ -60,43 +55,19 @@ export class UserService {
...
-    return jwt.sign({ userId }, appConfig.jwtSecret, {
-      expiresIn,
-    });
-  }
-  verifyToken(token: string): jwt.JwtPayload | string | boolean {
-    if (!appConfig.jwtSecret) throw Error("Invalid token secret");
+  async refreshToken(token: string): Promise<string> {
     try {
-      return jwt.verify(token, appConfig.jwtSecret);
+      const { userId } = verifyToken(token) as IgenerateToken;
```
Comment (anshs-webelight) [inline server/services/user.service.ts]: why do we need to re-verify refreshToken? we are verifying this refreshToken in authToken middleware also, right?

Comment (anshs-webelight) [reply server/services/user.service.ts]: if you want userId from the token you can use `decode()` function from jsonwebtoken no need to verify token again

File: server/utils/bcrypt.ts (hunk 1 above)
Comment (Deepak-webelight) [reply server/utils/bcrypt.ts]: fixed

File: server/services/user.service.ts (hunk 2 above)
Comment (Deepak-webelight) [reply server/services/user.service.ts]: Fixed

**Comment by Deepak-webelight** [comment]
- bycrypt.ts file updated 
- implemented decode()

**Pull request merged**